sql/nm_spawn_points.sql

```diff
@@ -1989,6 +1989,7 @@ INSERT INTO `nm_spawn_points` VALUES (17596507,47,-132.983,0.000,178.993);
 INSERT INTO `nm_spawn_points` VALUES (17596507,48,-140.219,0.000,170.706);
 INSERT INTO `nm_spawn_points` VALUES (17596507,49,-92.879,0.000,218.895);
 INSERT INTO `nm_spawn_points` VALUES (17596720,0,-155.000,19.000,244.000);
+INSERT INTO `nm_spawn_points` VALUES (17596628,1,158,0,20);
```

Comment:
I also need to add MM's drop list to my mob_droplist as well i noticed
unsure how to edit without resubmitting the file